### ISSUE TITLE
Targeting players

### ISSUE THREAD

**Issue by Zomis** [Cardshifter/Cardshifter#102]
As more spells are added, it needs to be possible for the client to explicitly target the opponent player. Sometimes you might have three possible targets, for example you (the player), your opponent (the player), and a creature on the battlefield. In such cases, it should be possible to choose all three of these. Client currently does not support explicitly targeting a player.

**Comment by Phrancis** [comment]
I think this is working with the JSON cards implementation now? Also related, #213

**Comment by Zomis** [comment]
Nope, this does still not work. In the JavaFX GUI you cannot target your own player. Imagine there's a "Target player draws ten cards" spell, depending on the situation you might want to play that in yourself or in your opponent (as opponent takes damage from drawing from an empty deck). So you should have the ability to choose the player.
